Sofa/Component/Constraint/Lagrangian/Solver/src/sofa/component/constraint/lagrangian/solver/GenericConstraintSolver.cpp

```diff
@@ -94,6 +95,10 @@ GenericConstraintSolver::GenericConstraintSolver()
     , current_cp(&m_cpBuffer[0])
     , last_cp(nullptr)
 {
+    sofa::helper::OptionsGroup m_newoptiongroup(2,"PGS","NNCG");
```

Comment:
I think it was in https://github.com/sofa-framework/sofa/pull/2875 that we decided that the options names must be explicit, and avoid acronyms